app/models/program.rb

```diff
@@ -163,6 +163,23 @@ def any_event_for_this_date?(date)
     EventSchedule.where(schedule: selected_schedule).where(start_time: parsed_date..(parsed_date + 1)).any?
   end
 
+  ##
+  # Provides backwards compatibility for when the program had one cfp
+  #
+  # ====Returns
+  # * +ActiveRecord+ -> The program's cfp with cfp_type == 'events'
+  def cfp
+    return nil if cfps.for_events.blank?
+    cfps.for_events
+  end
+
+  ##
+  # ====Returns
+  # * +Array+ -> The types of cfps for which a cfp doesn't exist yet
+  def available_cfp_types
```

Comment:
I would rename this to something that is easier to associate with the result we get (the remaining types for which we can create a cfp)